applications/crossbar/doc/ref/ledgers.md

```diff
@@ -99,3 +99,13 @@ curl -v -X GET \
     http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/ledgers/{SOURCE_SERVICE}/{LEDGER_ID}
 ```
 
+## Fetch
```

Comment:
@mk1s This the ref file generated by `cb_api_endpoints`. When this file is change you have to manually update the actual doc directly under doc folder `applications/crossbar/doc/ledgers.md`.

So copy this new add section to to doc file, add appropriate title (Get summary by account to breakdown). Also if you can add a sample curl request and crossbar response.